### PR TITLE
adding easyconfigs: GHC-9.2.2-x86_64.eb

### PR DESCRIPTION
**Pull request by jkuhl-uni**
Hi there, this is an update to the GHC easy config.

### CODE REVIEW

**Comment by smoors** [comment]
@jkuhl-uni thanks a lot for your contribution!
you might be interested in the easybuild-github integration to make PRs easier, see here for more info http://tutorial.easybuild.io/2023-eb-eessi-uk-workshop/easybuild-contributing/#github-integration-features

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/g/GHC/GHC-9.2.2-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+version = '9.2.2'
+versionsuffix = '-x86_64'
+
+homepage = 'https://haskell.org/ghc/'
+description = """The Glorious/Glasgow Haskell Compiler"""
+
+toolchain = SYSTEM
+
+builddependencies = [
+    ('binutils', '2.32'),
```
Comment (smoors) [inline easybuild/easyconfigs/g/GHC/GHC-9.2.2-x86_64.eb:26]: @jkuhl-uni the failing test suggests that you have to add `M4` as a build dependency.
```suggestion
    ('binutils', '2.32'),
    ('M4', '1.4.19'),
```

Comment (jkuhl-uni) [reply easybuild/easyconfigs/g/GHC/GHC-9.2.2-x86_64.eb:26]: Hi @smoors,
sure, I'll push the changes in a few seconds.
Concerning the github integration: I actually used that feature, I think the token I am using is not the right one. I'll keep that in mind next time. Thank you for the link :)

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @jkuhl-uni!

**Pull request merged**